mbuild/pattern.py

```diff
@@ -130,6 +130,18 @@ def apply_to_compound(self, guest, guest_port_name='down', host=None,
 
 class Random2DPattern(Pattern):
     def __init__(self, n, seed=None, **kwargs):
+        """ Generate n random points on a 2D grid whose
```

Comment:
Could you get this into a one-line descriptor a la numpydoc style? http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Looks good otherwise, thank you!